### PR TITLE
Adds component to make ajax call for arbitrary endpoints

### PR DESCRIPTION
**Pull request by diogob**

### CODE REVIEW

File: src/c/admin-external-action.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            };
+
+            const updateItem = (res) => {
+                _.extend(item, res[0]);
+                complete(true);
+                error(false);
+            };
+
+            const submit = () => {
+                console.log('options on submit', builder.requestOptions.url);
```
Comment (vicnicius) [inline src/c/admin-external-action.js]: Let's remove those logs.

File: src/c/admin-external-action.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+            const reload = _.compose(builder.model.getRow, h.idVM.id(item[builder.updateKey]).parameters),
+                  l = m.postgrest.loader(builder.requestOptions, m.request);
+
+            const reloadItem = (data) => {
+                reload().then(updateItem);
+            };
+
+            const requestError = (err) => {
+                console.log('err', err);
```
Comment (vicnicius) [inline src/c/admin-external-action.js]: should we remove this log here?

Comment (vicnicius) [inline src/c/admin-external-action.js]: Actually it does make sense to log the error. x) So we should only remove the requestOptions log, right?

File: src/c/admin-external-action.js (hunk 1 above)
Comment (vicnicius) [inline src/c/admin-external-action.js]: Removed on master.

Comment (diogob) [inline src/c/admin-external-action.js]: Thanks ;)

**Pull request merged**